docs/ru/main/base/s3/dev/s3-golang/s3-golang-bucket/s3-golang-bucket.meta.json

```
@@ -0,0 +1,12 @@
+{
+  "title": "Операции с бакетами",
+  "pageDescription": "",
+  "shortDescription": "Работа с бакетами S3",
+  "metaTitle": "Операции с бакетами - VK Cloud",
```

здесь и в аналогичных местах metaTitle: убрать "- VK Cloud", сейчас этот заголовок подгружается автоматически

```suggestion
  "metaTitle": "Операции с бакетами",
```